modules/get-started/pages/quick-start.adoc

```diff
@@ -36,111 +55,343 @@ docker compose version
+// (step {"action":"saveScreenshot", "path": "js-filter.png", "directory": "../../modules/console/images", "overwrite": "byVariation", "maxVariation": 10})
++
+image::console:js-filter.png[]
+
+. Replace the default JavaScript code with the following:
++
+[,js]
 ----
-[+] Running 4/4
-⠿ Network redpanda_network                 Created  0.0s
-⠿ Volume "redpanda-quickstart_redpanda-0"  Created  0.0s
-⠿ Container redpanda-0                     Started  0.3s
-⠿ Container redpanda-console               Started  0.6s
+return value.email.includes(".edu");
 ----
---
-Three Brokers::
+
+. Click *Save* to apply the filter.
+// (step {"action":"find", "selector": "[data-testid='add-javascript-filter-save']", "matchText": "Save", "click": true})
 +
---
+You should see only messages that include the specific domain.
+
+By default, the filter runs on the newest 50 messages. You can run the filter on older messages by changing the start offset. For example, to start from the oldest messages, select *Beginning*.
+// (step {"action":"goTo", "url": "http://localhost:8080/topics/transactions"})
+// (step {"action":"find", "selector": ".chakra-tabs__tablist button[data-index='5']", "matchText": "Documentation", "click": true})
+// (step {"action":"wait"})
+// (step {"action":"saveScreenshot", "path": "topic-documentation.png", "directory": "../../modules/console/images", "overwrite": "byVariation"})
+
+=== Schema Registry
+
+On the *Schema Registry* page, you see an overview of your schemas. You can create, manage, and inspect your schemas without leaving Redpanda Console.
+
+// (step {"action":"find", "selector": "[data-testid='nav-link-Schema Registry']", "matchText": "Schema Registry", "click": true})
+// (step {"action":"goTo", "url": "http://localhost:8080/schema-registry/subjects/transactions?version=latest"})
+// (step {"action":"wait"})
+// (step {"action":"saveScreenshot", "path": "schema-reg.png", "directory": "../../modules/console/images", "overwrite": "byVariation"})
+
+image::console:schema-reg.png[]
+
+See also: xref:console:ui/schema-reg.adoc[].
+
+=== Configure access control for Redpanda
+
+Managing access control lists (ACLs) can be complex, but Redpanda Console simplifies this with an intuitive interface.
 
-. xref:get-started:attachment$three-brokers/docker-compose.yml[Download] the following `docker-compose.yml` file on your local file system.
+This quickstart deployment includes a bootstrapped superuser with full administrative privileges. The superuser has unrestricted access to all topics, consumer groups, transactions, and cluster-level operations. By default, the credentials for the superuser are:
+
+- Username: `superuser`
+- Password: `secretpassword`
+
+For improved security and role-based access control, you should create additional users with restricted permissions as needed.
+
+On the *Security* page, you can:
```

Comment:
@JakeSCahill can you add a test for this page name? The **Security** page is likely getting renamed when RBAC in Cloud goes GA in 1-2 months. Right now it sounds like it'll be renamed **IAM: Data plane** (because in Cloud, there will be a separate **IAM: Control plane** page).